examples/basic-wallet/src/lib.rs

```diff
@@ -23,8 +23,9 @@ use bindings::exports::miden::basic_wallet::*;
 
 bindings::export!(MyAccount with_types_in bindings);
 
-use miden::{Asset, NoteType, Recipient, Tag};
+use miden::{component, Asset, NoteType, Recipient, Tag};
 
+#[component]
```

Comment:
Per my other comment, I think we should just expect users to add `#[derive(Default)]`, and document that we expect these types to implement `Default`. It'll be a lot clearer to readers of the code then that the trait is implemented for this type - not that I can think of a scenario where they would be manually instantiating it, but just more as a general Rust practice I guess.

Reply:
See my comment above.